builtin/settingtypes.txt

```diff
@@ -1881,12 +1882,11 @@ world_aligned_mode (World-aligned textures mode) enum enable disable,enable,forc
 #    Warning: This option is EXPERIMENTAL!
 autoscale_mode (Autoscaling mode) enum disable disable,enable,force
 
-#    When using bilinear/trilinear/anisotropic filters, low-resolution textures
-#    can be blurred, so automatically upscale them with nearest-neighbor
-#    interpolation to preserve crisp pixels. This sets the minimum texture size
-#    for the upscaled textures; higher values look sharper, but require more
-#    memory. Powers of 2 are recommended. This setting is ONLY applied if
-#    bilinear/trilinear/anisotropic filtering is enabled.
+#    When using bilinear/trilinear filtering, low-resolution textures
```

Comment:
iirc from the texture filtering discussion we found out that on some GPUs, enabling aniso automatically/always results in bilinear being applied too (e.g. https://github.com/minetest/minetest/issues/14007#issuecomment-1816989090)

=> aniso should actually also trigger texture_min_size to be applied to avoid blurriness for these GPUs

Reply:
this is just me correcting the docs, I didn't decide this.
either way texture_min_size is terrible hack and we should get rid of it rather than enable it more often.

> some GPUs #14007

this GPU is now clearly out of our support range and I'm hoping this case was "just" a broken driver

Reply:
> this is just me correcting the docs, I didn't decide this.

ok, the documentation was wrong before. did some digging, this behavior is quite old: introduced in 6be6fb78a4becc9bf4e999ff5aaa3c2d75fcc428. the wrong documentation was introduced in 1da73418cd2ea0e03e8289f54a47dededcf8b331.

> this GPU is now clearly out of our support range and I'm hoping this case was "just" a broken driver

ok

Reply:
I suggest moving the documentation changes to a separate commit to make history digging easier in the future

Reply:
I suggest moving the documentation changes to a separate commit to make history digging easier in the future